app/assets/stylesheets/base/_forms.scss

```diff
@@ -0,0 +1,33 @@
+.container * {
+  box-sizing: border-box;
+  font-size: 20px;
+}
+
+.simple_form {
+  border: 1px solid grey;
+  padding: 10px;
+  list-style-type: none;
+  max-width: 800px;
+  margin: 0 auto;
+  background-color: #dce0f2;
```

Comment:
Alphabetic order for the css definitions?

Reply:
👌